config/default.ts

```diff
@@ -1,7 +1,11 @@
-export default ({ 
-    APIGenerated: {
-      port: process.env.PORT || 3000,
-      enforceWhitelist: process.env.WHITELIST || "FALSE",
-      mainnet: process.env.MAINNET || "FALSE",
-    }
-  } as ConfigType);
\ No newline at end of file
+export default {
+  contract: {
+    nodeUrl: process.env.CONTRACT_HOST || "http://localhost:8545",
+    chainId: process.env.CHAIN_ID || "1000"
+  },
+  APIGenerated: {
```

Comment:
I don't understand the word `Generated` here at all. Can we try to find a better name for this key too?

Reply:
I'm not sure about this either. cc @SebastienGllmt @nicarq

Reply:
not sure what it's referring to either

Reply:
I'm not sure how to call it. It's basically the parameters for the API